vagrant-kubernetes-istio/localregistry.yaml

```diff
@@ -1,82 +0,0 @@
-apiVersion: v1
```

Comment:
The reference of this file is in https://github.com/istio/istio/blob/master/tests/util/localregistry/localregistry.yaml.

Reply:
There are a couple of differences between 2 files:
1) value: kube-registry instead of kube-registry.kube-system.svc.cluster.local for REGISTRY-HOST value
2) Updated resources value for memory and cpu

Didn't update the existing file as it works for minikube as the readme in it's folder mentions...